src/common/pure/CurrencyLogo/hooks/useCurrencyLogoURIs.ts

```diff
@@ -3,10 +3,10 @@ import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
 import EthereumLogo from 'legacy/assets/images/ethereum-logo.png'
 import XDaiLogo from 'legacy/assets/cow-swap/xdai.png'
-import { useExternalTokenLogo } from 'common/hooks/useExternalTokenLogo'
 import { NATIVE_CURRENCY_BUY_ADDRESS } from 'legacy/constants'
 import uriToHttp from 'lib/utils/uriToHttp'
 import { ADDRESS_IMAGE_OVERRIDE } from 'legacy/constants/tokens'
+import { useProxyTokenLogo } from '../../../../api/proxy'
```

Comment:
why not absolute import?

Reply:
My IDE generates it. It's a reason why I added the new rule to eslint :)